gempython/tools/amc_user_functions_xhal.py

```diff
@@ -800,7 +807,7 @@ def programAllOptohybridFPGAs(self, maxIter=5, ohMask=None):
     def readADCsMultiLink(self, adcDataAll, useExtRefADC=False, ohMask=None, debug=False):
         """
         Reads the ADC value from all unmasked VFATs
-        adcDataAll - Array of type c_uint32 of size 24*12=288
+        adcDataAll - Array of type c_uint32 of size self.NVFAT*12=288
```

Comment:
this can't be correct still, since you've added a potentially different constant value to the equation

Reply:
Oups, oversight =) Thanks!